doc/whats-new.rst

```diff
@@ -102,6 +102,7 @@ Internal Changes
 - Adds :py:func:`open_datatree` into ``xarray/backends`` (:pull:`8697`) By `Matt
   Savoie <https://github.com/flamingbear>`_.
 
+- Adds ``treenode`` functionality into ``xarray/core`` (:pull:`8757`) By `Matt Savoie <https://github.com/flamingbear>`_.
```

Comment:
@TomNicholas You want some acknowledgment that this is moving your code?  Maybe change Adds to migrates or include your name here?

Reply:
I don't really need acknowledgement but I guess adding my name there might help someone later who wants to know who to blame for a bug they find? 😁

Reply:
hopefully `git blame -C -C -C` will help out a lot but I'll update these.